lms/static/sass/course/layout/_reset_deadlines.scss

```diff
@@ -1,23 +1,51 @@
-div.reset-deadlines-banner {
-  background-color: theme-color("primary");
-  display: none;
-  flex-wrap: wrap;
-  padding: 15px 20px;
-  margin-top: 5px;
+.dates-banner {
+    border-radius: 4px;
+    border: solid 1px #9cd2e6;
+    background-color: #eff8fa;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    padding: 24px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    max-width: $text-width-readability-max;
 
-  div,
-  button {
-    flex: 0 1 auto;
+    .dates-banner-text {
+        font-size: 16px;
+        line-height: 24px;
+        color: #414141;
+    }
 
-    &.reset-deadlines-text {
-      color: theme-color("inverse");
-      margin: 10px 10px 10px 0;
+    .banner-has-button {
+        flex: 1 1 20em;
+        max-width: 70%;
     }
 
-    &.reset-deadlines-button {
-      color: #0075b4;
-      background-color: theme-color("inverse");
-      cursor: pointer;
+    .upgrade-button {
+        align-self: start;
+        flex: none;
+
+        a {
+            text-decoration: none;
+        }
+
+        button {
+            display: block;
+            border-radius: 2px;
+            border: solid 1px #0175b4;
+            background: white;
+            color: #2d323e;
+            font-size: 14px;
+            font-weight: bold;
+            line-height: 24px;
+            padding: 7px 18px;
+
+            &:hover,
+            &:focus,
+            &:active {
+              cursor: pointer;
+              box-shadow: 0 2px 1px $shadow;
+            }
+        }
     }
   }
-}
```

Comment:
That feels weird -- to end the file on an indented brace?

Reply:
oh thats my bad, this isnt intended